packages/main/src/webComponents/Wizard/Wizard.mdx

```diff
@@ -13,7 +13,114 @@ import { excludePropsForAbstract } from '@sb/utils';
 
 ## Example
 
-<Canvas of={ComponentStories.Default} />
+<Canvas of={ComponentStories.Default} sourceState="none" />
+
+<details>
+
+<summary>Show Code</summary>
+
+```tsx
+const WizardComponent = () => {
+  const [selected, setSelected] = useState('1');
+  const [disabled, setDisabled] = useState<Record<string, boolean>>({ '2': true, '3': true, '4': true });
+  const [hidden, setHidden] = useState<Record<string, boolean>>({ step3Btn: true, finalizeBtn: true });
+  const goToStep2 = () => {
+    setDisabled((prev) => {
+      const { '2': _omit, ...rest } = prev;
+      return rest;
+    });
+    setSelected('2');
+  };
+  const goToStep3 = () => {
+    setDisabled((prev) => {
+      const { '3': _omit, ...rest } = prev;
+      return rest;
+    });
+    setSelected('3');
+  };
+  const handleStep2Completed = (e) => {
+    if (e.target.checked) {
+      setHidden((prev) => {
+        const { step3Btn: _omit, ...rest } = prev;
+        return rest;
+      });
+    }
+  };
+  const finalizeWizard = () => {
+    alert('Wizard is now completed!');
+  };
+  const handlePriceInput = (e) => {
+    if (e.target.value) {
+      setHidden((prev) => {
+        const { finalizeBtn: _omit, ...rest } = prev;
+        return rest;
+      });
+    }
+  };
+
+  return (
+    <Wizard style={{ height: '400px' }}>
+      <WizardStep icon={productIcon} titleText="Product Type" selected={selected === '1'}>
```

Comment:
```suggestion
      <WizardStep icon={productIcon} titleText="Product Type" selected={selected === 1}>
```

Reply:
With my initial commit I forgot to update the React state when the user clicks on a step, since I'm now using a dataset to pass the respective step `string` is better here as then there is no parsing necessary.

Reply:
Yes, this is connected to first suggestion, if first suggestion will be valid, then all of them are valid, if not then not